sklearn/utils/__init__.py

```diff
@@ -254,6 +254,10 @@ def resample(*arrays, **options):
         generator; If None, the random number generator is the RandomState
         instance used by `np.random`.
 
+    stratify : ndarray or None (default=None)
```

Comment:
Since you added the check_array, this is now `array-like` (you can pass a list for instance)